presto-parser/src/main/java/com/facebook/presto/sql/tree/SliceLiteral.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import com.google.common.base.Preconditions;
+import io.airlift.slice.Slice;
+
+import java.util.Objects;
+
+import static com.google.common.io.BaseEncoding.base64;
+
+public class SliceLiteral
```

Comment:
I don't think we need this class, if we make the change I suggested in `LiteralInterpreter`, and this is a little hacky. Let me know if you run into issues making the change to LiteralInterpreter